src/main/scala/org/phenoscape/kb/ingest/bgee/BgeeExpressionToOWL.scala

```diff
@@ -25,7 +25,8 @@ object BgeeExpressionToOWL {
 
   def translate(expressionLine: String): Set[OWLAxiom] = {
     val items = expressionLine.split("\t", -1)
-    if (items.length == 1 || items(6).startsWith("absent") || items(6).startsWith("Expression")) {
+    
+    if (items.length == 1 || items(6).startsWith("absent") || items(6).startsWith("Expression") || items(2).startsWith("ZFA")) {
```

Comment:
@calliem could you add a comment here that we are skipping ZFA terms because these are all embryonic or otherwise pre-adult developmental stages that wouldn't make sense with the data the KB is otherwise populated with? Otherwise this reason may be lost when someone looks at it in the future.